e2e/playwright/support/ui/pages/boards_view.ts

```diff
@@ -21,6 +24,10 @@ export default class BoardsViewPage {
         this.topHead = page.locator('.top-head');
         this.editableTitle = this.topHead.getByPlaceholder('Untitled board');
         this.shareButton = page.getByRole('button', {name: '󰍁 Share'});
+        this.duplicateBoardButton = page.getByLabel('Duplicate board');
+        this.deleteBoardButton = page.getByLabel('Delete board');
```

Comment:
This is a good candidate for a Boards menu component in the sidebar but I'm fine as is. We can move in the future.

Reply:
noted Saturn